miden/src/cli/data.rs

```diff
@@ -201,6 +206,12 @@ impl InputFile {
                         .map_err(|e| format!("failed to parse a Sparse Merkle Tree: {e}"))?;
                     merkle_store.extend(tree.inner_nodes());
                 }
+                MerkleData::PartialMerkleTree(data) => {
+                    let entries = Self::parse_partial_merkle_tree(data)?;
+                    let tree = PartialMerkleTree::with_leaves(entries)
+                        .map_err(|e| format!("failed to parse a Partial Merkle Tree: {e}"))?;
+                    merkle_store.extend(tree.inner_nodes());
```

Comment:
One other thing that could be interesting here (and for the other cases above) is to print out the root of the Merkle structure added to the Merkle store. For example:
```
Added Partial Merkle tree with root {root} to the Merkle store
```
This will help user figure out which root can be used in their program.